plf-tomcat-resources/src/main/resources/conf/catalina.properties

```diff
@@ -19,8 +19,7 @@
 # passed to checkPackageAccess unless the
```

Comment:
What is the patched config here comparing to tomcat 8 default file (The diff is made with Tomcat 7 patched file)